README.rst

```diff
@@ -64,6 +64,15 @@ Install dependencies and the ``sysl`` package with symlinks ::
 
   > pip install pytest flake8 -e .
 
+Set up your environment for diagram generation by providing `PlantUML <http://plantuml.com/>` 
```

Comment:
i'm not familiar with `rst` but perhaps this should be ` Link PlantUML <http://plantuml.com/>`

http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks

Reply:
I'm not that familiar with rst the either as one can see, it was meant to be a link. In order to stay consistent with the rest of this doc I'm using the `` `Link text<URL>`_`` syntax.
Done.